ops/deploy-ccs.groovy

```diff
@@ -63,34 +63,10 @@ def findAmis() {
 			'Name=state,Values=available' --region us-east-1 --output json | \
 			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
 		).trim(),
-		bfdPipelineAmiId: sh(
-			returnStdout: true,
-			script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=bfd-amzn2-jdk17-etl-??????????????' \
-			'Name=state,Values=available' --region us-east-1 --output json | \
-			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
-		).trim(),
-		bfdServerAmiId: sh(
-			returnStdout: true,
-			script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=bfd-amzn2-jdk17-fhir-??????????????' \
-			'Name=state,Values=available' --region us-east-1 --output json | \
-			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
-		).trim(),
-		bfdMigratorAmiId: sh(
-			returnStdout: true,
-			script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=bfd-amzn2-jdk17-db-migrator-??????????????' \
-			'Name=state,Values=available' --region us-east-1 --output json | \
-			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
-		).trim(),
-		bfdServerLoadAmiId: sh(
-			returnStdout: true,
-			script: "aws ec2 describe-images --owners self --filters \
-			'Name=name,Values=server-load-??????????????' \
-			'Name=state,Values=available' --region us-east-1 --output json | \
-			jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
-		)
+		bfdPipelineAmiId: '',
+		bfdServerAmiId: '',
+		bfdMigratorAmiId: '',
+		bfdServerLoadAmiId: ''
```

Comment:
Shoot. I must not have been clear. While this specific effort under BFD-2212 does represent value for a subset of cases, it cannot be used to replace the existing deployment process by itself. So, this `findAmis()` method needs to remain unchanged until the whole _soon-to-be-but-not-yet-replaced_ deployment process is replaced.

Reply:
I reverted it.